new/collect/add-all-package-jsons.ts

```diff
@@ -0,0 +1,21 @@
+import type { Module } from "../types";
+import { sync as glob } from "glob";
+import { yellow } from "chalk";
+
+import { logger } from "../helpers/logger";
+
+const { log } = logger("add all package.json");
+
+const ROOT = process.cwd();
```

Comment:
Isn't `ROOT` already in the `env` object somewhere? Would make sense I guess - so that it's identical everywhere.

Reply:
good catch. I'll remove it